doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-rest-api-requests-proc.adoc

```diff
@@ -84,6 +84,9 @@ For REST client:
 * *HTTP Headers*: Set the following headers:
 ** `Accept`: `application/json`
 ** `Content-Type`: `application/json`
++
+NOTE: When you add `field=not_null` in the `Content_Type`, then the null fields are excluded from the REST API response.
```

Comment:
typo, "fields" instead of "field"
```suggestion
NOTE: When you add `fields=not_null` in the `Content_Type`, the null fields are excluded from the REST API response.
```

Reply:
Done, fixed it